packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -19,7 +19,7 @@ Requires: ruby
 BuildRequires: ruby
 BuildRequires: rubygems-devel
 BuildRequires: rubygem(deface)
-BuildRequires: (rubygem(fog-proxmox) >= 0.15 with rubygem(fog-proxmox) < 1)
+BuildRequires: (rubygem(fog-proxmox) >= 0.15.1 with rubygem(fog-proxmox) < 0.16)
```

Comment:
This is not in 3.14 yet

Reply:
Should I create PR for fog-proxmox packaging for 3.14 first?

Reply:
Here is fog-proxmox PR for rpm/3.14: https://github.com/theforeman/foreman-packaging/pull/11817

Reply:
Our process can't handle a `BuildRequires` update in the same PR so that must be merged first. I missed that the first time round, so 2 separate PRs is the way to go :(

Reply:
Here is PR for fog-proxmox: https://github.com/theforeman/foreman-packaging/pull/11818 and I updated this PR to only have cherry-pick of foreman_fog_proxmox.